app/views/subnets/index.html.erb

```diff
@@ -7,7 +7,7 @@
   <thead>
     <tr>
       <th class="col-md-3"><%= sort :name, :as => s_("Subnet|Name") %></th>
-      <th class="col-md-2"><%= sort :network, :as => s_("Subnet|Network") %></th>
+      <th class="col-md-2"><%= sort :network, :as => s_("Subnet|Network Address") %></th>
       <th class="col-md-3"><%= _("Domains") %></th>
```

Comment:
Don't create a second string (`Subnet|Network Address`) without removing the first (`Subnet|Network`), either change the existing translation or remove the original. (Not that attrs should be removed from the generated model attrs file.)